src/utils/notification.js

```diff
@@ -1,33 +1,43 @@
 import Notiflix from 'notiflix';
 
-const notifyUserRegisterSuccess = () => {
-  Notiflix.Report.success(
-    'Registration Success!',
-    'Now You have an account in Wallet.',
-    'Have fun!',
-  );
+const notifyUserEmailSenTSuccess = email => {
+  Notiflix.Report.success('Verification email sent!', `Check :${email} Inbox.`, 'OK!');
 };
 
-const notifyRegistrationFailure = () => {
+const notifyUserEmailSenTFailure = message => {
   Notiflix.Report.failure(
-    'Registration Fail!ed.',
-    'This User already exist or too short Username / e-mail / password (each must have at least 7 characters).',
+    'Sending the verification email failed.',
```

Comment:
A nie lepiej po prostu "Registration failed" ?

Reply:
Nie, bo to nie jest sam proces rejestracji. To jest weryfikacja majla.